airflow/providers/amazon/aws/hooks/redshift.py

```diff
@@ -15,10 +15,19 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""Interact with AWS Redshift, using the boto3 library."""
+"""Interact with AWS Redshift, using the boto3 or redshift_connector library."""
```

Comment:
```suggestion
"""Interact with AWS Redshift clusters"""
```

(It's not really that relevant to the users what libraries we use.

Reply:
fixed in ea5b4c8dbe93388639ac9c0d2f02ac4c4a9f7bfd